research/gaia/gremlin/gremlin_core/src/result_process.rs

```diff
@@ -95,8 +95,16 @@ fn property_to_pb(result_property: &ResultProperty) -> result_pb::TagEntries {
             let mut props_pb = vec![];
             for (prop_name, prop_val) in value_map {
                 let pb_value = object_to_pb_value(prop_val);
-                let property =
-                    result_pb::Property { key: prop_name.to_string(), value: Some(pb_value) };
+                // TODO: should be prop_name or prop_id
+                let property = match prop_name {
+                    PropKey::Str(prop_name) => {
+                        result_pb::Property { key: prop_name.to_string(), value: Some(pb_value) }
+                    }
+                    PropKey::Id(prop_id) => {
```

Comment:
Better add a TODO and mark as `unimplemented!()`

Reply:
Done